src/pages/AddGame.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import dispatchCreateGame from '../store/slices/create_game_slice';
+import dispatchGetUser from '../store/slices/get_user_slice';
+import style from '../assets/components_styles/add_game.module.css';
+
+const AddGame = () => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const [userInfo, setUserInfo] = useState(JSON.parse(localStorage.getItem('userInfo')));
+  const [game, setGame] = useState({
+    name: '',
+    description: '',
+    icon: '',
+    price: 0,
+    category: '',
+    owner_id: userInfo.id,
+  });
+
+  const handleInputChange = (event) => {
+    setGame({
+      ...game,
+      [event.target.name]: event.target.value,
+    });
+  };
+
+  const createGame = async (e) => {
+    e.preventDefault();
+    const data = await dispatchCreateGame(dispatch, game);
+    if (!data.error) {
+      const user = await dispatchGetUser(dispatch, userInfo.username);
+      setUserInfo(user);
+      window.alert('Game created successfully!');
```

Comment:
- [ ] **(OPTIONAL)** I don't like using the `window` object to alert users using the browsers functionality, this could cause some problems depending on what's happening in the app and I believe it's a bad practice. Instead, a modal with a message would be a lot nicer. Of course, this is not mandatory as it's just my opinion.